hail/src/main/scala/is/hail/expr/ir/Emit.scala

```diff
@@ -13,7 +13,7 @@ import is.hail.expr.ir.streams.{EmitStream, StreamProducer, StreamUtils}
 import is.hail.io.{BufferSpec, InputBuffer, OutputBuffer, TypedCodecSpec}
 import is.hail.io.fs.FS
 import is.hail.linalg.{BLAS, LAPACK, LinalgCodeUtils}
-import is.hail.types.{TypeWithRequiredness, VirtualTypeWithReq, tcoerce}
+import is.hail.types.{tcoerce, TypeWithRequiredness, VirtualTypeWithReq}
```

Comment:
I'm not sure why we're still seeing these kinds of change @patrick-schultz - I thought we had run scalafix on the whole codebase?

Reply:
Ah, looks like I forgot to add a scalafix check in the `check-hail` make target. I'll PR. I just tried running scalafix locally and this was the only failure that has slipped in.

Reply:
https://github.com/hail-is/hail/pull/14276